### ISSUE TITLE
lessThan reports error if compared to empty field

### ISSUE THREAD

**Issue by enboig** [jquery-validation/jquery-validation#2463]
## Your environment
* Version of `jquery-validate`: v1.19.3
* Browser name and version: firefox 109.0.1

## Current behavior

When using rule `lessThan` comparing to an empty field, error is triggered.

## Expected behavior

Error should only be triggered if both fields are populated; user could/should add `required` to the other field if necessary.

## Live demo 
QUnit.test( "form(): with lessThan", function( assert ) {
	assert.expect( 2 );
	var form = $( "#testForm5" )[ 0 ],
		v = $( form ).validate();

	$( "#x1, #x2" ).val( "hi" );
	assert.ok( v.form(), "Valid form" );
	$( "#y2" ).val( "" );
	assert.ok( v.form(), "Valid form" );
} );

**Comment by bytestream** [comment]
I'm not sure if that's a bug, and it could also be considered a breaking change if changed..

**Comment by enboig** [comment]
I agree it could be a breaking change...
For now I will create my custom functions `lessThanIfOtherNotEmpty`; next to `lessThanEqualNumberIfOtherNotEmpty` to compare float numbers correctly.

**Comment by enboig** [comment]
One question about my custom methods is if I should create a class (like does `.validate-lessThanEqual-blur` for each of it).
```js
j.validator.addMethod(
    "lessThanIfOtherNotEmpty",
    function (val1, element, param) {
        var target = j(param);

        // if (
        //     this.settings.onfocusout &&
        //     target.not(".validate-lessThanIfOtherNotEmpty-blur").length
        // ) {
        //     target
        //         .addClass("validate-lessThanIfOtherNotEmpty-blur")
        //         .on("blur.validate-lessThanIfOtherNotEmpty", function () {
        //             j(element).valid();
        //         });
        // }
        var val2 = target.val(),
            normalizer;
        if ((rules = target.rules())) {
            if (typeof rules.normalizer === "function") {
                normalizer = rules.normalizer;
            } else if (typeof this.settings.normalizer === "function") {
                normalizer = this.settings.normalizer;
            }
            // If normalizer is defined, then call it to retreive the changed value instead
            // of using the real one.
            // Note that `this` in the normalizer is `element`.
            if (normalizer) {
                val2 = normalizer.call(element, val2);

                // Delete the normalizer from rules to avoid treating it as a pre-defined method.
                delete rules.normalizer;
            }
        }

        if (val1 == "" || val2 == "") {
            return true;
        }

        return val1 < val2;
    },
    "Please enter a lesser value."
);

```
